tests/rbd/rbd_utils.py

```diff
@@ -1409,8 +1413,22 @@ def client_config_read_only(rbd, client_id, pool, **kw):
     osd_role = kw.get("osd_role", "allow *")
     cmd = f"ceph auth caps {client_id}"
     cmd += f" mon '{mon_role}' mds '{mds_role}'"
-    cmd += f" mgr '{mgr_role} pool={pool}"
-    if kw.get("namespace"):
-        cmd += f" namespace={kw['namespace']}"
-    cmd += f"' osd '{osd_role}'"
+    cmd += f" mgr '{mgr_role}'"
+    cmd += f" osd '{osd_role}'"
     return rbd.exec_cmd(cmd=cmd)
+
+
+def save_client_config_keyring(rbd, client_id, **kw):
```

Comment:
Ideally this should be in global cephci utility
Please check once whether it could be possible to move it to global utility

cc @manasagowri

Reply:
What do you mean by global utility Vasi? Which file exactly?